website/tables/bitbucket/bitbucket_workspaces.md

```diff
@@ -0,0 +1,20 @@
+# Table: bitbucket_workspaces
+
+https://developer.atlassian.com/cloud/bitbucket/rest/api-group-workspaces/#api-group-workspaces
+
+The primary key for this table is **uuid**.
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_id|`uuid`|
+|_cq_parent_id|`uuid`|
+|uuid (PK)|`uuid`|
+|name|`utf8`|
+|type|`utf8`|
+|slug|`utf8`|
+|is_private|`bool`|
+|created_on|`timestamp[ms, tz=Z]`|
+|updated_on|`timestamp[ms, tz=Z]`|
```

Comment:
I wonder why this is `timestamp[ms, tz=Z]` instead of `timestamp[ms, tz=UTC]` that we normally get in the docs 🤔

Reply:
https://github.com/apache/arrow/blob/4b7d699ab9ee/go/arrow/datatype_fixedwidth.go#L819

Reply:
@candiduslynx That definition seems to have `UTC` in it, why is it `Z` in the generated docs here?

Reply:
@hermanschaaf because almost all of our plugins use Go's `Timestamp_us` while in Java SDK we might just skip the timezone?

Reply:
Although we use UTC there, too:
https://github.com/cloudquery/plugin-sdk-java/blob/main/lib/src/main/java/io/cloudquery/scalar/Timestamp.java#L16-L17
https://github.com/cloudquery/plugin-sdk-java/blob/main/lib/src/main/java/io/cloudquery/transformers/TypeTransformer.java#L43-L44

Reply:
`Z` comes from the `ZoneOffset.UTC`, though.
I wonder if it's something of an issue

Reply:
Running the CLI table conversion through a debugger, I can see the `Z` string from Java makes it all the way though to the data type creation [here](https://github.com/apache/arrow/blob/main/go/arrow/ipc/metadata.go#L947) in the CLI process. The timezone is just a string in Go with no additional validation (although `Z` is a valid timezone).

Reply:
Ah cool, that makes sense now. No worries in that case, I think we can live with the minor discrepancy in the docs. Thanks for looking into it!